secretsConfig.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-config:
+appConfig:
```

Comment:
This is a fix for a bug from previous changes. Maybe tests against main.go could include using the provided example config.